vispy/color/tests/test_color.py

```diff
@@ -344,4 +344,9 @@ def test_normalize():
     assert_allclose([y.min(), y.max()], [0.2975, 1-0.2975], 1e-1, 1e-1)
 
 
+def test_colormap_creation():
```

Comment:
If you are already here, do you mind adding some more tests for the different parametrizations?